amico/util.py

```diff
@@ -22,9 +23,14 @@ def fsl2scheme( bvalsFilename, bvecsFilename, schemeFilename = None, bStep = 1.0
     if schemeFilename is None:
         schemeFilename = os.path.splitext(bvalsFilename)[0]+".scheme"
 
+    if kwargs and 'delmimiter' in kwargs:
```

Comment:
Great idea to be more flexible!

Just one little note: for readability, could this block be replaced by:
```python
if kwargs :
    delimiter = kwargs.get('delimiter')
```
If then `delimiter` is not defined in `kwargs`, the function `get` would return `None`.